### ISSUE TITLE
Downgrade versions command broken

### ISSUE THREAD

**Issue by Axel29** [netz98/n98-magerun2#487]
### Preconditions

1. Magento >= 2.3
2. n98-magerun2 latest version

### Steps to reproduce

1. Install Magento from 2.3.0 at least
2. Run `./n98-magerun2.phar sys:setup:downgrade-versions` command

### Expected result

1. Modules with higher database versions than code should be downgraded

### Actual result

1. Exception triggered: 

![image](https://user-images.githubusercontent.com/3015249/65615625-f68eac80-dfb9-11e9-9551-bf924880b590.png)

```
[BadMethodCallException]                               
  Module 'Magento_Store' data-version is not available.  
```

### Additional information

The problem is that since Magento 2.3.0, a lot of Magento modules lost their data-version and schema-version and thus the command can't downgrade anymore.

**Comment by ktomk** [comment]
I was not able to confirm the issue, but:

Would some command-line parameter to hint the Magento 2.3 situation be useful in your opinion?i

Not to say this is a viable solution, just according to the  additional information given, the command might be in the need of branching before/after 2.3 which could be done by such an option (or implicit).

**Comment by cmuench** [comment]
One of my colleagues told me that Magento has now an automatic downgrade logic in the core.
Maybe it's available since a 2.3.x version.
Nevertheless, we should handle modules without setup version which is possible since the XML configuration of schemas is available.

**Comment by cmuench** [comment]
A fix will be released with v4.0.0 the next days.
